pyaedt/edb_core/materials.py

```diff
@@ -838,7 +827,7 @@ def get_property_by_material_name(self, property_name, material_name):
 
     @pyaedt_function_handler()
     def add_material_from_aedt(self, material_name):
-        """Add a material read from ``syslib amat`` library.
+        """Add a material read from ``syslib AMAT`` library.
```

Comment:
```suggestion
        """Add a material read from the ``syslib AMAT`` library.
```
Here, ``syslib AMAT`` is tagged as a code entity. If this is a file name, the case should exactly match the case of the file name.